portal-api/src/v1/controllers/facilities.controller.js

```
@@ -117,6 +117,8 @@ const queryAllFacilities = async (filters = {}, sort = {}, start = 0, pagesize =
           exporter: '$deal.exporter.companyName',
           // exporter in lowercase for sorting
           lowerExporter: { $toLower: '$deal.exporter.companyName' },
+          // Facility stage - specifically used for Risk expired
+          facilityStage: '$facilityStage',
```

Just a thought -- if we only want devs to use this for risk expired, should we throw an error if someones searching for something other than risk expired to stop misuse?